Shared/Tests.Sync.Shared/PermissionsTests.cs

```diff
@@ -47,5 +50,35 @@ public void PermissionChange_ShouldNotBeInDefaultSchema()
         {
             Assert.That(RealmSchema.Default.Find(nameof(PermissionChange)), Is.Null);
         }
+
+        [Test, Explicit]
```

Comment:
Are we going to also have tests about permissible state changes depending on state of server?